### DISCUSSION TITLE
Microsoft Teams

### DISCUSSION THREAD

**Discussion by nik-rev** [catppuccin/catppuccin#2395] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

Can be themed via user styles just like the other 86 websites.

Lots of organizations use Microsoft Teams and it would be quite nice to have a theme for it.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by AlwaysNur** [comment]
Made a [pull request](https://github.com/catppuccin/userstyles/pull/1979)
